code/modules/mob/living/carbon/xenomorph/castes/warrior/warrior.dm

```diff
@@ -75,3 +83,36 @@
 	if(ishuman(AM))
 		return
 	..()
+
+// ***************************************
+// *********** Primordial procs
+// ***************************************
+///Handles primordial warrior empowered abilities, returns TRUE if the ability should be empowered.
+/mob/living/carbon/xenomorph/warrior/proc/empower()
```

Comment:
Should be a proc on xenomorph, with this code in the ovveridden version for warrior